dizoo/classic_control/mountain_car/config/mtcar_rainbow_config.py

```diff
@@ -0,0 +1,58 @@
+from easydict import EasyDict
+
+# DI-Engine uses EasyDict for configuration, by convention
+mtcar_rainbow_config = EasyDict(dict(
+    exp_name='mtcar_rainbow_seed0',
+    env=dict(
+        collector_env_num=8,
+        evaluator_env_num=5,
+        n_evaluator_episode=5,
+        stop_value=195,
```

Comment:
modify proper stop_value

Reply:
Config file will be updates on branch "mtcar_rainbow" in #471. Will reflect these notes in #471 as well.